client/my-sites/upgrades/checkout-thank-you/custom-domain-purchase-detail.jsx

```diff
@@ -12,22 +12,24 @@ import { hasCustomDomain } from 'lib/site/utils';
 
 const CustomDomainPurchaseDetail = ( { selectedSite, hasDomainCredit, translate } ) => {
 	const renderClaimCustomDomain = () =>
-		<PurchaseDetail
-			icon="globe"
-			title={ translate( 'Select Your Custom Domain' ) }
-			description={
-				translate(
-					'Your plan includes a free custom domain. Replace {{em}}%(siteDomain)s{{/em}} ' +
-					'with a custom domain to personalize your site.',
-					{
-						args: { siteDomain: selectedSite.domain },
-						components: { em: <em /> }
-					}
-				)
-			}
-			buttonText={ translate( 'Claim your free domain' ) }
-			href={ `/domains/add/${ selectedSite.slug }` }
-		/>;
+		selectedSite.plan.user_is_owner
```

Comment:
Thanks for the ping.

> do you think it'd be worth it to connect these leaf components?

I think (and as we discussed a few times), we should always use selectors for retrieving data from Redux state. However, I don't think this change warrants connecting the whole component yet. So, instead of grabbing `user_is_owner` out of `selectedSite.plan`, let's use a selector for this in the outside connected component and pass it down to the `CustomDomainPurchaseDetail` component.

What do you think @jeremeylduvall and @gwwar?